public/content/documentation/web/component/number-input.md

```diff
@@ -0,0 +1,219 @@
+      - I HEAR it identifies itself as an editable input
+      - I HEAR hints or errors are read after the label, related inputs include a group name (Ex: Enter your personal information)
+      - I HEAR if applicable, it expresses its state (required, disabled / dimmed / unavailable)
+   - THEN when I use the number keys I HEAR numbers are entered
+   - THEN when I use non-number keys I HEAR nothing is entered
+
+3. Mobile screenreader
+
+   - WHEN I use a mobile screenreader (Talkback, VoiceOver)
+      - AND I swipe to focus on a number input
+      - I HEAR its purpose is clear
+      - I HEAR it identifies itself as an editable input
+      - I HEAR hints or errors are read after the label, related inputs include a group name (Ex: Enter your personal information)
+      - I HEAR if applicable, it expresses its state (required, disabled / dimmed / unavailable)
+   - THEN when I enter a number I HEAR the numeric keypad is revealed
+
+Full information: [https://www.magentaa11y.com/MagentaA11yV2#/web-criteria/component/number-input](https://www.magentaa11y.com/MagentaA11yV2#/web-criteria/component/number-input)
+
+## Code examples
+
+### Don't use `type="number"`
+
+The `type="number"` input is intended for **integers** and includes features we _don't_ want (like stepper/scroll functionality) that is a nuisance to everyone 
+
+- Phone, credit card, pin etc. are not integers
+- NVDA doesn't fully support `type="number"` inputs at this time
+
+### Use `type="text"` for number inputs
+
+Use `type=text` with `inputmode="numeric"` with an input pattern and JS to filter out non-numeric characters.
+
+```html
+<label for="pin">
+  Pin number
+</label>
+<input type="text"
+       id="pin"
+       aria-describedby="hint"
+       inputmode="numeric"
+       pattern="[0-9]*">
+<div id="hint" class="hint">
+  The pin number will expire after 1 hour
+</div>
+```
+<!-- TODO for all examples, this is allowing letters AND numbers despite pattern -->
+<example>
+   <label for="pin">
+   Pin number
+   </label>
+   <input type="text"
+         id="pin"
+         aria-describedby="hint"
+         inputmode="numeric"
+         pattern="[0-9]*">
+   <div id="hint" class="hint">
+   The pin number will expire after 1 hour
+   </div>
+</example>
+
+### Disabled and focusable number input (preferred)
+
+- Using the `aria-disabled` attribute will allow the input to be focusable and more discoverable
+
+```html
+<label for="security-id">
+  Security ID number
+</label>
+<input type="text"
+       id="security-id"
+       aria-describedby="security-id-hint"
+       inputmode="numeric"
+       pattern="[0-9]*"
+       aria-disabled="true">
+<div id="security-id-hint" class="hint">
+  The Security ID number will expire after 1 hour
+</div>
+```
+
+<example>
+   <label for="security-id">
+   Security ID number
+   </label>
+   <input type="text"
+         id="security-id"
+         aria-describedby="security-id-hint"
+         inputmode="numeric"
+         pattern="[0-9]*"
```

Comment:
It's weird how this is allowing letters to be input but the old site's example does not allow it. 

Maybe we can add a TODO to come back to it.

Reply:
Never mind. I spaced on the fact that the type=text will allow it. I guess there's some JS in the old site not permitting letters.

Reply:
I added a TODO because it's supposed to act disabled as well but it is not. I think I will actually comment out this particular demo as it is not working well